views/imagestream-body.html

```diff
@@ -36,6 +36,8 @@
     <dd ng-if-end><tt>{{imagestream.spec.dockerImageRepository}}</tt></dd>
     <dt>Pulling repository</dt>
     <dd><tt>{{imagestream.status.dockerImageRepository}}</tt></dd>
+    <dt ng-if-start="imagestream.status.publicDockerImageRepository">Public pulling repo</dt>
+    <dd ng-if-end><tt>{{imagestream.status.publicDockerImageRepository}}</tt></dd>
```

Comment:
@spadgett not sure if it'd be better to leave this out if there is no `publicDockerImageRepository` field, or simply state that there is none / instructions on how expose a registry?